libphidgets/CMakeLists.txt

```diff
@@ -1,11 +1,18 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(libphidgets)
 
-find_package(catkin REQUIRED COMPONENTS rosbuild roslib mk rosbuild rospack)
+find_package(catkin REQUIRED COMPONENTS mk rospack)
```

Comment:
just tested it: `pkg_check_module` does not work for `libusb-dev` as well...
Now I'll definitely leave it like this ;-)